.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```

Comment:
Can the `.idea` folder be ignored in `.gitignore` and then can you remove the `.idea` dir from here?